src/button/internal.tsx

```diff
@@ -221,7 +227,22 @@ export const InternalButton = React.forwardRef(
     const buttonContent = (
       <>
         <LeftIcon {...iconProps} />
-        {shouldHaveContent && <span className={clsx(styles.content, analyticsSelectors.label)}>{children}</span>}
+        {shouldHaveContent && (
+          <>
+            <span className={clsx(styles.content, analyticsSelectors.label)}>{children}</span>
+            {external && (
+              <>
+                &nbsp;
+                <span
+                  role="img"
+                  aria-label={i18n('i18nStrings.externalIconAriaLabel', i18nStrings?.externalIconAriaLabel)}
+                >
+                  <Icon name="external" className={testUtilStyles['external-icon']} />
+                </span>
+              </>
+            )}
+          </>
+        )}
         <RightIcon {...iconProps} />
```

Comment:
What happened to a thread about printing a warning when this is used together with the external icon?